packages/testing/README.md

```diff
@@ -41,6 +53,29 @@ This will have the following side effect:
   - use the plugin [chai-dom-equals](https://www.npmjs.com/package/@open-wc/chai-dom-equals)
   - enables cleanup after each test for `fixture` and `litFixture`
 
+### Manual Karma Setup
```

Comment:
Should these next two `###` be a part of `### Manual Setup` above?

Reply:
made it to ####